float32 altitude

---
bool success